### ISSUE TITLE
Can't execute first step for npm install

### ISSUE THREAD

**Issue by nqwang** [hdorgeval/testcafe-starter#3]
After cloning this repo, and try to execute npm install, it will pop up an error like below.
```
> testcafe-starter@2.1.0 preinstall C:\Users\nqwang\Documents\github\testcafe-starter
> rimraf node_modules

'rimraf' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! testcafe-starter@2.1.0 preinstall: `rimraf node_modules`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the testcafe-starter@2.1.0 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\nqwang\AppData\Roaming\npm-cache\_logs\2018-10-26T06_05_13_257Z-debug.log
```
The reason is that computer doesn't have `rimraf` installed before executing `npm install`. It is better to update readme file to add this command `npm install rimraf -g` before `npm install`.

**Comment by hdorgeval** [comment]
Hi @nqwang , thank you for your excellent feedback. I will update asap the project to get rid of `rimraf`, because I think installation of a project should never depend on a globally installed package.

**Comment by hdorgeval** [comment]
@nqwang : I keep this issue opened until project is updated. Thanks again for this point.

**Comment by nqwang** [comment]
@hdorgeval It's my pleasure.

**Comment by hdorgeval** [comment]
Hi @nqwang , this issue has been solved in v3.1.0
